### DISCUSSION TITLE
Is there a way to change the naming convention in batch converting?

### DISCUSSION THREAD

**Discussion by VNDDNV** [mifi/lossless-cut#1872] [Ideas]
Hi, Just wondering is there a way to change the naming convention in mass/batch converting feature?
At the moment all batch converted files end with "html5ified-fastest-audio".
So for example "cat.ts" will be "cat-html5ified-fastest-audio.mp4".
Tried looking everywhere for a setting so it can end in "2" like in single covert file mode but couldn't see any.

**Comment by mifi** [comment]
hi. that's not possible no. one of the reasons for the somewhat obscure naming is to avoid the chance of accidentally overwriting existing files
